### ISSUE TITLE
'Client' object has no attribute 'get_historical_klines'

### ISSUE THREAD

**Issue by hsachdevah** [sammchardy/python-binance#165]
Not able to fetch historical klines
Getting the error above when I call the function

AttributeError: 'Client' object has no attribute 'get_historical_klines'

**Comment by AntoineGS** [comment]
By any chance is it possible Client should be client? (lowercase). It sounds like Client is not referring to the correct object.

**Comment by sammchardy** [comment]
Hi @hsachdevah please check the documentation about creating the client instance.
https://python-binance.readthedocs.io/en/latest/overview.html#initialise-the-client

```
from binance.client import Client
client = Client(api_key, api_secret)
```

Then use the lowercase `client` instance to make calls, e.g.

```
client.get_historical_klines(symbol='TRXETH', interval=Client.KLINE_INTERVAL_4HOUR, start_str='1 year ago')
```

**Comment by jazzypan** [comment]
> Hi @hsachdevah please check the documentation about creating the client instance.
> https://python-binance.readthedocs.io/en/latest/overview.html#initialise-the-client
> 
> ```
> from binance.client import Client
> client = Client(api_key, api_secret)
> ```
> 
> Then use the lowercase `client` instance to make calls, e.g.
> 
> ```
> client.get_historical_klines(symbol='TRXETH', interval=Client.KLINE_INTERVAL_4HOUR, start_str='1 year ago')
> ```

Not working...

**Comment by nerod10** [comment]
1. pip uninstall python-binance
2. pip install python-binance

**Comment by AlexChiang0208** [comment]
I have same question, but after I install again "pip install python-binance", it was solved!!

**Comment by nahyun-kim-lab** [comment]
I uninstalled and installed again, and used the same code as above, but still has the same error. AttributeError: 'Client' object has no attribute 'get_historical_klines'

**Comment by nahyun-kim-lab** [comment]
Same for a lot of methods.. 

AttributeError: 'Client' object has no attribute 'get_recent_trades'
AttributeError: type object 'Client' has no attribute 'KLINE_INTERVAL_30MINUTE' (while running .get_klines()

But others work such as get_order_book or get_ticker

**Comment by nahyun-kim-lab** [comment]
I have the same error, many methods - but not all - were returning AttributeErrors. I tried both uninstalling and the code above to no avail. I thought I had solved this issue by deleting and creating a new API, but no.*

* Should anybody have a problem with the API Key, I believe Binance changed the format slightly, so just create a new one, and it solved it for me. 

Here are some of my errors. 
AttributeError: 'Client' object has no attribute 'get_avg_price'
AttributeError: 'Client' object has no attribute 'get_historical_klines'

**Comment by 369Martin369** [comment]
> 1. pip uninstall python-binance
> 2. pip install python-binance

yes this work. terrible, there are so many binance packages available, but "python-binance" verison 1.0.16" works!

**Comment by RiiNagaja** [comment]
I had everything working for almost 2 years, but recently upgraded to Python 3.11 . Now I face the same issues as listed here.
- AttributeError: 'Client' object has no attribute 'get_asset_balance'
- Or all the constants not being recognized despite python-binance listing that Client.CONSTANT is the right way to do it.

Reinstalling the module was not the issue, but I went and installed python 3.9 instead (which is better for many other uses like machine learning anyway). That fixed it, so python-binance is likely just incompatible with some more modern python versions.
